evap/staff/views.py

```diff
@@ -1386,6 +1386,36 @@ def evaluation_email(request, evaluation_id):
     )
 
 
+def import_or_copy_participants(request, operation, import_type, evaluation, copy_form) -> bool:
+    if "test" in operation:
+        return False
+
+    if "replace" in operation:
+        deleted_person_count, deletion_message = helper_delete_users_from_evaluation(evaluation, operation)
+
+    if "import" in operation:
```

Comment:
I think we should just not pass `operation` and instead have `replace: bool` and `action: ImportAction` (or similar) where `ImportAction` is an enum with variants `Copy` and `Import`